monkey-runner/build.gradle

```diff
@@ -0,0 +1,24 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = versions.gradleWrapperVersion
+}
+
+buildscript {
+    apply from: 'dependencies.gradle'
+
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath gradlePlugins.gradle
+        classpath gradlePlugins.gradleAndroidCommandPlugin
+        classpath gradlePlugins.bintrayRelease
+    }
+}
+
+apply from: 'dependencies.gradle'
```

Comment:
@xrigau I actually believe that is a related but slightly different issue, that can be resolved by introducing `sample/bulidSrc` module for instance.
The reason why apply here doesn't work is because the `buildscript` configuration of the root project configuration is evaluated while the properties extension is not yet part of the model, therefore the error.